appveyor.yml

```diff
@@ -6,15 +6,22 @@ branches:
 
 cache:
   # Note that we don't use cache for local Maven repository - see https://github.com/SonarSource/sonar-java/pull/525
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-  - C:\bin\apache-maven-3.2.5 -> appveyor.yml
+  - C:\Users\appveyor\maven -> appveyor.yml
```

Comment:
In sonar-csharp, we're not caching Maven itself: see https://github.com/SonarSource/sonar-csharp/blob/master/appveyor.yml

But then, sonar-csharp gets build much less frequently than sonar-java, so stability issues are harder to spot